src/compiler.jl

```diff
@@ -8552,7 +8593,24 @@ function GPUCompiler.codegen(output::Symbol, job::CompilerJob{<:EnzymeTarget};
         primal_job = CompilerJob(primal, config2, job.world) # TODO EnzymeInterp params, etc
     end
 
-    mod, meta = GPUCompiler.codegen(:llvm, primal_job; optimize=false, toplevel=toplevel, cleanup=false, validate=false, parent_job=parent_job, ctx)
+    ctx = nothing
+    c = nothing
+
+    @static if VERSION < v"1.9-"
```

Comment:
Why do we need this? Re below can we not get the corresponding context from mod?

E.g. ctx = LLVM.context(mod)